app/components/TopNav.vue.ts

```diff
@@ -36,9 +37,17 @@ export default class TopNav extends Vue {
   @Inject() incrementalRolloutService: IncrementalRolloutService;
 
   slideOpen = false;
-  studioModeTooltip = 'Studio Mode';
+
+  studioModeTooltip = $t('Studio Mode');
+  settingsTooltip = $t('Settings');
+  helpTooltip = $t('Get Help');
+  logoutTooltip = $t('Logout');
+  sunTooltip = $t('Day mode');
+  moonTooltip = $t('Night mode');
+
   availableChatbotPlatforms = ['twitch', 'mixer', 'youtube'];
 
+
```

Comment:
Delete this extra line, as TSLint will complain about it.  Otherwise good to go.

Reply:
Removed it. Thanks for letting me know as I didn't realize that was there!